src/neo4j_genai/experimental/pipeline/pipeline.py

```diff
@@ -47,7 +47,7 @@
 
 
 class RunStatus(enum.Enum):
-    UNKNOWN = "UNKNOWN"
+    UNKNOWN = None
```

Comment:
Why set this to `None`? Will it not get set to `"UNKNOWN"` any more?

Reply:
It was because the store returns `None` when the component has not started yet. But now I realize it would better handled in the `get_status_for_component` method, I'll go with that.

Reply:
https://github.com/neo4j/neo4j-genai-python/pull/123/commits/5fc686bf8574b7b6d1450e42d7157eb44ed25513

I've also removed the statuses that are not used (should have done it a long time ago)